designer/server/plugins/session.ts

```diff
@@ -0,0 +1,25 @@
+import config from "../config";
+import yar from "yar";
+import { ServerRegisterPluginObject } from "@hapi/hapi";
+
+export const configureYarPlugin = (): ServerRegisterPluginObject<yar> => {
+  return {
+    plugin: yar,
+    options: {
+      cache: {
+        expiresIn: config.sessionTimeout,
+      },
+      cookieOptions: {
+        password:
+          config.sessionCookiePassword ||
```

Comment:
we are not setting this on runner too , so it will not be an issue if we dont make changes to our deployments
but we have env variables to override if required